website/src/styles/tokens.css

```
@@ -0,0 +1,230 @@
+  --dark-chrome: #100f11; /* nav / footer chrome */
+  --dark-faq: #161518; /* faq / feature section */
+  --dark-hairline: rgba(255, 255, 255, 0.07);
+  --text-on-dark-heading: #f7f6f4;
+  --text-on-dark-body: rgba(255, 255, 255, 0.6);
+  --text-on-dark-faint: rgba(255, 255, 255, 0.45);
+
+  /* ── typography.css ───────────────────────────────────────── */
+  --font-display: "GT Alpina", Georgia, "Times New Roman", serif;
+  --font-bitmap: "PP Mondwest", "Courier New", monospace;
+  --font-sans:
+    "Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)